src/datasets/main_ping_tables.md

```diff
@@ -0,0 +1,21 @@
+# Main Ping Tables
+
+As described in the [pipeline schemas deployment docs](https://docs.telemetry.mozilla.org/concepts/pipeline/schemas.html),
+data engineering has a process of generating schemas for pings and deploying them to BigQuery. The Main Ping table (`telemetry.main`)
+is one of those generated tables.
+
+## Problems Querying the Main Ping Table: `telemetry.main`
+
+Because we generate a schema with all of the probes for the main ping, the number of columns has grown by a large amount: at this point,
+there are over 10k columns. When querying this table, it causes [BQ to scan a very large number of files](https://console.cloud.google.com/support/cases/detail/25679061?project=moz-fx-data-shared-prod).
+An end-user will see this as a _very_ slow query. Queries remain slow even when filtering on clustered columns, like `sample_id`.
+
+To reduce the time for querying main ping data, we have included two new tables: `telemetry.main_1pct`, and `telemetry.main_nightly`.
```

Comment:
```suggestion
To reduce the time for querying main ping data, we have included two new tables: `telemetry.main_1pct`, and `telemetry.main_nightly`. These can return results for simple queries in a matter of seconds where a logically equivalent query on `telemetry.main` may take minutes.
```

I want to make the benefits more concrete here rather than using language like "very slow query".